scripts/pyjoke

```diff
@@ -39,7 +39,14 @@ def main():
         parser.error(str(exc.message))
         exit(-1)
 
-    joke = get_joke(language=args.language, category=args.category)
+    try:
+        joke = pyjokes.get_joke(language=args.language, category=args.category)
+    except pyjokes.LanguageNotFoundError:
+        print('No such language %s' % args.language)
+        exit(-1)
+    except pyjokes.CategoryNotFoundError:
+        print('No such category %s' % args.category)
```

Comment:
`'There are no jokes for category {}'.format(args.category)`. ??
Well, you are the one immersed in the RP English. You should know. :smiley: